applications/endoscopy_tool_tracking/CMakeLists.txt

```diff
@@ -36,5 +36,7 @@ if(HOLOHUB_DOWNLOAD_DATASETS)
   )
 endif()
 
-add_subdirectory(cpp)
+if(NOT HOLOHUB_WHEEL_TESTING)
+    add_subdirectory(cpp)
+endif()
```

Comment:
Control flow for core subdirectories should not depend on a TESTING variable. Need to consider rename and potential conditions that could invalidate the condition, such as in the case of `multiai_ultrasound` depending on C++ operators.